src/java/org/apache/cassandra/service/accord/serializers/IVersionedWithKeysSerializer.java

```diff
@@ -323,44 +277,132 @@ private void serializeLargeSubset(AbstractRanges serialize, int serializeCount,
             }
         }
 
+        public Routables<?> deserializeSubsetInternal(Routables<?> superset, DataInputPlus in) throws IOException
```

Comment:
I am a bit confused about why this is unused now

Reply:
this particular caller now uses some of the internal methods this method calls directly. But, I have another patch in flight that may invoke these methods directly, and anyway they may be useful for the future so would prefer not to delete them here.